search-admin/config/deploy.rb

```diff
@@ -0,0 +1,25 @@
+set :application, "search-admin"
+set :capfile_dir, File.expand_path('../', File.dirname(__FILE__))
+set :server_class, "backend"
+
+set :run_migrations_by_default, true
+
+load 'defaults'
+load 'ruby'
+load 'deploy/assets'
+
+set :source_db_config_file, 'to_upload/database.yml'
+
+set :config_files_to_upload, {
+  'to_upload/secrets.yml' => 'config/secrets.yml'
```

Comment:
Yup @issyl0 is correct, `secrets/to_upload/thing_to_upload`. As [an example](https://github.com/alphagov/govuk-app-deployment/blob/master/content-store/config/deploy.rb).